src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -1044,6 +1045,45 @@ def get_file_driver(self) -> Optional[ManagedClusterStorageProfileFileCSIDriver]
 
         return profile
 
+    def get_blob_driver(self) -> Optional[ManagedClusterStorageProfileBlobCSIDriver]:
+        """Obtain the value of storage_profile.blob_csi_driver
+
+        :return: Optional[ManagedClusterStorageProfileBlobCSIDriver]
+        """
+        enable_blob_driver = self.raw_param.get("enable_blob_driver")
+        disable_blob_driver = self.raw_param.get("disable_blob_driver")
+
+        if enable_blob_driver is None and disable_blob_driver is None:
+            return None
+
+        profile = self.models.ManagedClusterStorageProfileBlobCSIDriver()
+
+        if enable_blob_driver and disable_blob_driver:
+            raise MutuallyExclusiveArgumentError(
+                "Cannot specify --enable-blob-driver and "
+                "--disable-blob-driver at the same time."
+            )
+
+        if self.decorator_mode == DecoratorMode.CREATE:
+            if enable_blob_driver:
+                profile.enabled = True
+            else:
+                profile.enabled = False
```

Comment:
It should be None when customers create aks without `--enable-blob-driver`. But it will be true when creating aks with `--enable-blob-driver` set explicitly.

Reply:
Good catch! thank you.